duet/explores/mobile_android_country.explore.lkml

```diff
@@ -1,6 +1,8 @@
 include: "../views/*.view.lkml"
 
 explore: mobile_android_country {
+  label: "Mobile Android Funnel"
```

Comment:
I prefixed this with Mobile so that it will end up next to the iOS one. I also shortened the name so it fits on a single line.